lib/google/apis/options.rb

```diff
@@ -101,5 +104,6 @@ def merge(options)
     RequestOptions.default.skip_serialization = false
     RequestOptions.default.skip_deserialization = false
     RequestOptions.default.api_format_version = nil
+    RequestOptions.default.opencensus_span_name = "google-api-client"
```

Comment:
You wrote in the PR description (emphasis mine):

> If the OpenCensus gem is present, AND there is an OpenCensus span context present, AND OpenCensus instrumentation **is enabled in the request options**, then [enable OpenCensus] 

But it looks like the default is to enable it here, making it opt-out rather than opt-in in the options.

Reply:
Sorry I changed this again to `use_opencensus` because it looks like the span name is standardized now. But your point is still relevant. Yes, I'm intentionally making it opt-out. The argument is, if the application has opencensus in its bundle, then tracing is intended.